tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = clean, py35, py36, py37, stats, lint, docs
+envlist = clean, py35, py36, py37, py27, stats, lint, docs
```

Comment:
hmm.. why do we need py2?

Reply:
Not 100% sure, but I grabbed the task since I was already in the repo: https://strateos.atlassian.net/browse/STDEV-32

Reply:
I think the main issue was when we were using the txpy repo when Igor was only py2 compatible. However, now that its is both py2 and py3 compatible. I don't think we need it. I can roll this back when I chuck out these commits and changelog updates.